### DISCUSSION TITLE
antd 5 modal style is not supporting in safari

### DISCUSSION THREAD

**Discussion by bibinantony98** [ant-design/ant-design#43967] [Q&A] [answered]
antd 5 css not supported in Safari version 13

`:where` is not supported in Safari browser version 13

I added StyleProvider hashPriority="high" as in documents, but still, antd models are in  `:where`

Screenshot 2023-08-02 at 1 03 23 PM

**Comment by zombieJ** [comment, marked as answer]
Check your `@ant-design/cssinjs` version. It may not align with antd used one if `hashPriority` not working. And also check Static function do not consume context which also not work with StyleProvider.

**Reply by bibinantony98** [reply]
version is not the issue, my issue happened when I import antd modal outside Styleprovider, but I call the modal inside of it (I use a plugin folder for creating a common modal), when I switch the import back to component it worked
